stickyKeysHunter.sh

```diff
@@ -119,14 +119,27 @@ rm ${temp}
 # Some systems seemed to need a bit more time to load before they accepted input
 sleep 2
 
-# Send the shift key 5 times to trigger
-echo -e "${blue} Attempting to trigger sethc.exe backdoor"
-xdotool key --window ${window} shift shift shift shift shift
+# Send Windows key + p to trigger displayswitch.exe
+echo -e "${blue} Attempting to trigger displayswitch.exe backdoor"
+xdotool key --window ${window} super+p
+
+# Send Windows key + = to trigger magnifier.exe, also Windows Key + - to reverse magnify effect
```

Comment:
super+ PLUS ( aka super+EQUAL) will launch magnify.exe for Windows 8 and up. I also have it press super+minus to reverse the magnify effect if magnify.exe launches correctly